client/my-sites/media-library/list-no-content.jsx

```diff
@@ -85,7 +85,7 @@ class MediaLibraryListNoContent extends Component {
 					className="media-library__no-content-upload-button is-primary"
 					site={ this.props.site }
 				>
-					{ this.props.translate( 'Upload Media' ) }
+					{ this.props.translate( 'Upload media' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.